account_invoice_overdue_reminder/hooks.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 ForgeFlow, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(cr):
+    _logger.info(
+        "Pre-creating column overdue_reminder_last_date for table account_move"
+    )
+    cr.execute(
+        """
+        ALTER TABLE account_move
+        ADD COLUMN overdue_reminder_last_date Date;
```

Comment:
```suggestion
        ADD COLUMN IF NOT EXISTS overdue_reminder_last_date Date;
```
the same in the others